packages/connect/src/api/ethereum/api/ethereumGetAddress.ts

```diff
@@ -36,20 +39,9 @@ export default class EthereumGetAddress extends AbstractMethod<'ethereumGetAddre
             : this.payload;
 
         // validate bundle type
-        validateParams(payload, [
-            { name: 'bundle', type: 'array' },
-            { name: 'useEventListener', type: 'boolean' },
```

Comment:
`useEventListener` is defined in CommonParams, but it's true that it's not validated at runtime. I could probably put it inside `GetAddressSchema` since all the `GetAddress` methods use it

Reply:
Added to `GetAddress` in [a9a657e](https://github.com/trezor/trezor-suite/pull/10280/commits/a9a657ea7bec70e46e18a322d22b9c4f37745242)